assets/scss/6-layout/_header.scss

```diff
@@ -0,0 +1,34 @@
+.header {
+  padding: $space-medium 0 $space-base;
+  text-align: center;
+
+  @include media-query('medium') {
+    padding: $space-large 0 $space-medium;
+  }
+
+  &__logo {
+    color: brand-color('primary');
+    margin: 0 auto;
+    max-width: rem-calc(290);
+
+    @include media-query('medium') {
+      max-width: rem-calc(400);
+    }
+  }
+
+  // &__headline {
```

Comment:
same here, do we want to maintain this stuff commented? I am always a bit against commented code, unless there is a very strong reason.